recipe/meta.yaml

```diff
@@ -46,7 +46,7 @@ test:
 
 about:
   home: http://www.cpc.ncep.noaa.gov/products/wesley/wgrib2
-  license: GPL-2.0
+  license: GPL-2.0-only
```

Comment:
SPDX specifies either GPL-2.0-only or GPL-2.0-or-later. Honestly i'm not sure.

I dug up LICENSE-wgrib2 and i'm still not sure

```
9/2005, 11/2006, 9/2013, 6/2016, 12/2020

                    wgrib2's copyrights and licences

wgrib2 is comprised of components of varying copyrights and licences.


                               Libraries

aec (compression): BSD 2-clause license, DWD
gctpc (geolocation): public domain, USGS
g2clib (grib): public domain, NCEP/NOAA
HDF (used by netcdf4): HDF license
iplib (interpolation): public domain, NCEP/NOAA
ip2lib (interpolation): gnu, NCEP/NOAA
jasper (jpeg2000): IMAGE POWER JPEG-2000 PUBLIC LICENSE
libpng: libpng license
mysql: used if mysql is installed, see mysql code (not supplied)
netcdf: netcdf license, UCAR
openjpeg (jpeg2000): BSD 2-clause license, many
PROJ.4(geolocation): MIT style licence
zlib(compression): zlib license

Except for gctpc (public domain), all the above libraries are optional.
Of course, you may will loose some functionality.

                  wgrib2 *.c *.h *.sh and documentation


The components of wgrib2, the C/Fortran API and wgrib2 library are either in 
the public domain, covered by the GNU General Public License or covered by 
the Apache licence v2 (decenc_openjpeg.c).  No other licences will be accepted 
into this archive of wgrib2 modules.  (Already have too many licences for 
the libraries.)  

Distribution of wgrib2 source/executables needs to be compatible with all 
applicatable licences.


                         Contributing to wgrib2


Contributions of new files can either be public domain or GNU GPL.  If you 
modify an existing file, please state that your contribution is of the same 
(gnu or public domain) as the original file.  Otherwise it gets messy.  
```